src/components/Navbar.js

```diff
@@ -1,67 +1,78 @@
-import React from 'react'
-import logo_white from '../logo_white.svg'
+import React from "react";
+import { Link } from "react-router-dom";
+import logo_white from "../assets/logo_white.svg";
 
-import './Navbar.css'
+import "./Navbar.css";
 
 const MenuItems = [
   {
     title: "About",
-    target: "#About",
-    cName: "nav-link"
+    target: "About",
```

Comment:
```suggestion
    target: "/About",
```
Also do this for every other target in this array.

Reply:
Done